app/config/config_dev.yml

```diff
@@ -19,4 +19,19 @@ parameters:
 monolog:
   handlers:
     main:
+      type: stream
+      path: "%kernel.logs_dir%/%kernel.environment%.log"
       level: debug
+      channels: ["!event"]
+    # uncomment to get logging in your browser
+    # you may have to allow bigger header sizes in your Web server configuration
+    #firephp:
+    #    type: firephp
+    #    level: info
+    #chromephp:
+    #    type: chromephp
+    #    level: info
+    console:
+      type: console
+      process_psr_3_messages: false
+      channels: ["!event", "!doctrine", "!console"]
```

Comment:
not console?

Reply:
Looks a bit strange that the console logger doesn't log the console channel, but it's just the same term used for different things: The console channel is used for the Symfony Console component and has nothing to do with the log handler.